project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.4.6
+sbt.version=1.4.9
```

Comment:
ideally this would also have a corresponding change to the baseimage used for building